### PR TITLE
ICU-457 Updated mattermost-redux to fix teams not getting marked as read

### PR DESCRIPTION
**Pull request by hmhealey**
#### Ticket Link
https://mattermost.atlassian.net/browse/ICU-457

### CODE REVIEW

**Comment by enahum** [comment]
is this for 1.6? or was this included already? cause I did upgrade mattermost-redux for 1.6 I just need to merge the branch back to master

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "fuse.js": "^3.2.0",
     "intl": "1.2.5",
     "jail-monkey": "0.1.0",
-    "mattermost-redux": "mattermost/mattermost-redux",
+    "mattermost-redux": "mattermost/mattermost-redux#b9b0208da40407fb3c77875117e0687552466f02",
```
Comment (enahum) [inline package.json]: we shouldn't be including the commit here, only in the yarn.lock file

Comment (hmhealey) [reply package.json]: Why not? This is how we do it in the web app

**Comment by hmhealey** [comment]
@enahum This is for master only. It's too late for this to go into 1.6

File: package.json (hunk 1 above)
Comment (enahum) [reply package.json]: There is no need as yarn.lock already takes care of it so is redundant unless we plan on using npm, actually in the webapp shouldn't be necessary either

Comment (hmhealey) [reply package.json]: That's a good point. I've been either manually updating the hashes which is error prone or using `yarn add mattermost-redux#....` which kind of messes up the yarn.lock. I'll revert this line

**Comment by hmhealey** [comment]
The current version of Redux in this repo includes these changes

**Pull request closed without merging**